packages/html/src/pivotgrid/tests/pivotgrid-cell-states.tsx

```diff
@@ -1,94 +1,92 @@
-                                        <PivotGridCell as="th" colSpan={1} headerTitle="North America" />
-                                        <PivotGridCell as="th" rowSpan={1} toggle="down" headerTitle="Countries" headerRootCell />
-                                    </PivotGridRow>
-                                    <PivotGridRow hover selected>
-                                        <PivotGridCell as="th" colSpan={1} headerTitle="Europe" />
-                                        <PivotGridCell as="th" rowSpan={1} toggle="down" headerTitle="Countries" headerRootCell />
-                                    </PivotGridRow>
-                                    <PivotGridRow>
-                                        <PivotGridCell focus as="th" colSpan={1} headerTitle="Asia" />
-                                        <PivotGridCell focus as="th" rowSpan={1} toggle="down" headerTitle="Countries" headerRootCell />
-                                    </PivotGridRow>
-                                    <PivotGridRow>
-                                        <PivotGridCell selected as="th" colSpan={1} headerTitle="Africa" />
-                                        <PivotGridCell selected as="th" rowSpan={1} toggle="down" headerTitle="Countries" headerRootCell />
-                                    </PivotGridRow>
-                                    <PivotGridRow>
-                                        <PivotGridCell focus as="th" colSpan={2} headerTitle="Regions" headerRootCell headerTotalCell />
-                                        <PivotGridCell selected as="th" rowSpan={1} headerTitle="Countries" headerRootCell headerTotalCell />
-                                    </PivotGridRow>
-                                </PivotGridTbody>
-                            </PivotGridTable>
-                        }
-                    >
+                <PivotGridNormal
+                    emptyCell={
+                        <span className="k-pivotgrid-empty-cell k-focus" />
+                    }
+                    rowHeaders={
                         <PivotGridTable>
                             <colgroup>
-                                <col width="200" />
-                                <col width="200" />
-                                <col width="200" />
+                                <col />
+                                <col />
+                                <col />
                             </colgroup>
                             <PivotGridTbody>
                                 <PivotGridRow hover>
-                                    <PivotGridCell as="td" colSpan={1} content="1847.00" />
-                                    <PivotGridCell as="td" colSpan={1} content="1488.00" />
-                                    <PivotGridCell as="td" colSpan={1} content="3335.00" totalCell />
+                                    <PivotGridCell as="th" rowSpan={5} toggle="up" headerTitle="Regions" headerRootCell />
```

Comment:
In my opinion, when inside the body,\ we should use cells as "td" instead of "th"